agents/agent-writer-agent/actions/Sema4.ai/browsing/devdata/input_download_file.json

```diff
@@ -1,12 +1,19 @@
 {
     "inputs": [
         {
-            "inputName": "Download CSV file",
+            "inputName": "Download CSV file - will be over max filesize",
             "inputValue": {
                 "file_url": "https://mikahanninen.dev/data.csv",
                 "max_filesize_in_megabytes": 0,
                 "target_folder": ""
             }
+        },
+        {
+            "inputName": "Download CSV file",
+            "inputValue": {
+                "file_url": "https://mikahanninen.dev/data.csv",
```

Comment:
Is the data such that we could put it in the agent package..?
Calling anything like: `https://mikahanninen.dev` is a big no-no in enterprises.